legend-depot-entities-api/src/main/java/org/finos/legend/depot/services/api/entities/EntitiesService.java

```diff
@@ -52,4 +52,6 @@ default List<ProjectVersionEntities> getDependenciesEntitiesByClassifier(String
     {
         return getDependenciesEntitiesByClassifier(Arrays.asList(new ProjectVersion(groupId, artifactId, versionId)), classifier, transitive, includeOrigin);
     }
+
+    String resolveVersionId(String groupId, String artifactId, String versionId);
```

Comment:
does not seem like the correct place, plus we already have a function like so, can we think of something better